assets/css/channel.css

```diff
@@ -0,0 +1,59 @@
+#channel-banner-container {
+  margin-bottom: 1em;
+}
+
+#banner {
+  display: flex;
+  flex-direction: column-reverse;
+
+  height: calc(100vw / 6.2 - 1px);
+  background-size: cover;
+  background-position: center;
```

Comment:
I'm not sure what's the goal here? Why not letting the browser decide how to fit the banner using `<img>` and a parent `<div>`?